cmd/vcluster/cmd/start.go

```diff
@@ -138,6 +140,23 @@ func NewStartCommand() *cobra.Command {
 }
 
 func ExecuteStart(options *context2.VirtualClusterOptions) error {
+	// check the value of pod security standard
+	if options.PodSecurityStandard != "" {
+		found := func() bool {
```

Comment:
why create a function for this?

Reply:
To make sure that the passed argument is valid.